etc/packetbeat.yml

```diff
@@ -156,13 +159,16 @@ output:
     # IPv6 addresses should always be defined as: https://[2001:db8::1]:9200
     hosts: ["localhost:9200"]
 
-    # Optional protocol and basic auth credentials. These are deprectad
+    # Optional protocol and basic auth credentials. These are deprectated.
```

Comment:
This typo just comes from libbeat, right? So we have to fix it there and then call update again. Do I get it right?